### PR TITLE
Adding the option to download the file

### PR DESCRIPTION
**Pull request by AdarshdeepCheema**
Signed-off-by: Adarshdeep Cheema <adarshdeep.cheema@ibm.com>

This PR addresses Issue: https://github.com/zowe/zlux/issues/323
## PR Type
- [ ] Bug fix
- [x] Feature
- [ ] Other (Please indicate)

## PR Checklist
- [x] PR completes `npm run preCommit` without error
- [ ] Relevant Test cases have been added (Unit and or FVT)
- [x] Relevant update to CHANGELOG.md
- [ ] PR from forked repo? Ensure Allow edits by maintaners is set.

### CODE REVIEW

**Comment by AdarshdeepCheema** [comment]
Screen Shot 2022-01-20 at 8 40 35 AM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Change Log
 All notable changes to the USS-Explorer will be documented in this file.
 
+## <1.0.20>
```
[the remark this answers is not in the document]
Comment (AdarshdeepCheema) [reply CHANGELOG.md:4]: now it is 1.0.21

File: WebContent/js/actions/treeUSS.js [hunk 2]
```diff
@@ -172,6 +197,46 @@ export function createUSSResource(path, type) {
...
+            })
+            .then(response => {
+                if (response.ok) {
+                    return response.text();
+                }
+                return response.text().then(e => { throw Error(JSON.parse(e).message); });
+            })
+            .then(text => {
+                const blob = new Blob([text], { type: 'text/plain' });
+                if (window.navigator.msSaveOrOpenBlob) {
```
Comment (1000TurquoisePogs) [inline WebContent/js/actions/treeUSS.js:226]: which browsers support this? I had trouble finding doc on it other than mozilla saying its deprecated and non-standard. is this a function actually supported by the browsers Zowe supports, or is this like, internet explorer?

Comment (AdarshdeepCheema) [reply WebContent/js/actions/treeUSS.js]: This was for IE . I just found that IE is not in the supported browsers for ZOWE, I removed it now. 
Thanks for review

**Pull request merged**